driver/src/contracts/stablex_contract.rs

```diff
@@ -93,10 +85,12 @@ pub trait StableXContract {
         batch_index: U256,
         solution: Solution,
         claimed_objective_value: U256,
-    ) -> Result<()>;
+        gas_price: U256,
+        timeout: Option<usize>,
```

Comment:
Either explain in a comment or rename the variable so that the unit of the timeout is clear.